state/sequence_test.go

```diff
@@ -197,6 +198,15 @@ func (s *sequenceSuite) TestEnsureBackwards(c *gc.C) {
 	s.incAndCheck(c, s.State, "foo", 3)
 }
 
+func (s *sequenceSuite) TestResetSequenceOp(c *gc.C) {
```

Comment:
No need to test ops functions separately

Reply:
This has been reverted because we just can't insert reset Op into current Op slice.